test/system/post_test.rb

```diff
@@ -0,0 +1,32 @@
+require "application_system_test_case"
+# https://guides.rubyonrails.org/testing.html#implementing-a-system-test
+
+class PostTest < ApplicationSystemTestCase
+
+  def setup
+    activate_authlogic
+  end
+
+  test 'posting from the editor' do
+    visit '/'
+
+    click_on 'Login'
+    fill_in("username-login", with: "Bob")
+    fill_in("password-signup", with: "secretive")
+    click_on "Log in"
+
+    visit '/post'
+
+    fill_in("Title", with: "My new post")
+    fill_in("text-input", with: "All about this interesting stuff")
```

Comment:
Here, we need a better way to select the right field. Maybe this is because we're in wysiwyg and not markdown mode?

Reply:
lets try this: https://stackoverflow.com/questions/21436984/capybara-and-contenteditable#21691735